### PR TITLE
Groupby limitations

### PR DESCRIPTION
**Pull request by 1e-to**

### CODE REVIEW

File: sdc/datatypes/hpat_pandas_groupby_functions.py [hunk 1]
```diff
@@ -625,7 +625,13 @@ def sdc_pandas_series_groupby_var(self, ddof=1, *args):
...
     'method_name': 'count',
     'example_caption': 'Compute count of group, excluding missing values.',
-    'limitations_block': '',
+    'limitations_block':
+    """
+    Limitations
+    -----------
+    - This function may reveal slower performance than Pandas* on user system. Users should exercise a tradeoff
+    between staying in JIT-region with that function or going back to interpreter mode.
+    """,
```
Comment (Rubtsowa) [inline sdc/datatypes/hpat_pandas_groupby_functions.py]: Put this in a separate variable to avoid code duplication

Comment (1e-to) [reply sdc/datatypes/hpat_pandas_groupby_functions.py]: I do not think it is necessary here

Comment (Rubtsowa) [reply sdc/datatypes/hpat_pandas_groupby_functions.py]: @kozlov-alexey  what do you think?

Comment (1e-to) [reply sdc/datatypes/hpat_pandas_groupby_functions.py]: If we need to add or change limitations, we will have to rewrite the text from the variables back to the lines. To the fact that so clearly

Comment (densmirn) [reply sdc/datatypes/hpat_pandas_groupby_functions.py]: I vote for inserting the constant text `This function may reveal slower performance than Pandas* on user system. Users should exercise a tradeoff
    between staying in JIT-region with that function or going back to interpreter mode.` where it requires through string formatting:

```python
performance_limitation = `This function may reveal slower performance than Pandas* on user system. Users should exercise a tradeoff
    between staying in JIT-region with that function or going back to interpreter mode.`
...
'limitations_block':
    """
    Limitations
    -----------
    - {}.
    """.format(performance_limitation),
...
```

Comment (densmirn) [reply sdc/datatypes/hpat_pandas_groupby_functions.py]: or
```
performance_limitation = `This function may reveal slower performance than Pandas* on user system. Users should exercise a tradeoff
    between staying in JIT-region with that function or going back to interpreter mode.`
...
'limitations_block':
    f"""
    Limitations
    -----------
    - {performance_limitation}
    """,
...
```

**Comment by 1e-to** [comment]
> documentation based on these

image

**Pull request merged**